components/org.wso2.carbon.identity.branding.preference.resolver/src/main/java/org/wso2/carbon/identity/branding/preference/resolver/UIBrandingPreferenceResolverImpl.java

```diff
@@ -81,7 +81,7 @@
 public class UIBrandingPreferenceResolverImpl implements UIBrandingPreferenceResolver {
 
     private static final Log LOG = LogFactory.getLog(UIBrandingPreferenceResolverImpl.class);
-    private final ExecutorService executorService = Executors.newFixedThreadPool(1);
```

Comment:
We have changed the name in https://github.com/wso2/carbon-identity-framework/pull/5720.. So lets reflect that.

Reply:
fixed